README.md

```diff
@@ -160,13 +160,6 @@ Most of the commands need `nix` and will run only on Linux or macOS.
 - Windows 10/11
 - Daedalus testnet installation (similar version used in branch) in `C:\Program Files\Daedalus Testnet`
 - NodeJS 16
-- Python2 *
```

Comment:
These instructions is needed in case you want to run Daedalus on windows

Reply:
Hey, thanks 😄  I've just discussed with @danielmain and he's going to test that on Windows. Most probably it's no longer needed (as it was marked already* as node-sass dependency, which I've removed).

Reply:
We've decided to leave it as is for now (just without mentioning `node-sass` explicitly).